src/grasp_generator.cpp

```diff
@@ -1039,11 +1039,11 @@ bool GraspGenerator::generateSuctionGrasps(const Eigen::Isometry3d& cuboid_top_p
   double xy_increment = grasp_data->grasp_resolution_;
   double y_min = 0;
   double y_max =
-      (width + grasp_data->active_suction_range_y_) / 2.0 - grasp_data->suction_voxel_matrix_->voxel_y_width_;
+      (width + grasp_data->active_suction_range_y_) / 2.0 - grasp_data->grasp_resolution_;
```

Comment:
This could use xy_increment directly. Actually I thought that y_max and x_max should be a multiple of xy_increment or the 0 position wouldn't be covered, no?

Reply:
0 is always covered since it generates new poses by transforming existing grasps.